python/py-pylru/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pylru
+version             1.1.0
+revision            0
+categories-append   devel
+platforms           darwin
+license             GPL-2
+
+python.versions     27 34 35 36
+
+maintainers         @ipglider openmaintainer
```

Comment:
Your `maintainer` line should be of the form:

`maintainers         {somedomain.com:user @ipglider} openmaintainer`

Where "somedomain.com:user" is your "obscured" email address. See guide.macports.org for details.